lib/user_space/utrace.c

```diff
@@ -80,7 +84,8 @@ static int logbuf_map()
 		int available_bytes = sizeof trace_file_path -
 				      strlen(trace_file_path);
 		rc = snprintf(trace_file_path + strlen(trace_file_path),
-			available_bytes, "m0trace.%u", m0_pid_cached);
+			available_bytes, "m0trace.%u.%"PRIu32, m0_pid_cached,
```

Comment:
Let's just add a human-readable timestamp, see addb2/dump.c:_clock() for example.